src/main/java/plugily/projects/buildbattle/arena/ArenaRegistry.java

```diff
@@ -99,8 +98,8 @@ public boolean additionalValidatorChecks(ConfigurationSection section, PluginAre
     BaseArena baseArena = (BaseArena) arena;
 
     for(String plotName : plotSection.getKeys(false)) {
-      String minPointString = plotSection.getString(plotName + ".1", null);
-      String maxPointString = plotSection.getString(plotName + ".2", null);
+      String minPointString = plotSection.getString(plotName + ".minpoint", null);
+      String maxPointString = plotSection.getString(plotName + ".maxpoint", null);
 
```

Comment:
We are saving now 1 and 2 instead of min and max point

Reply:
Alright, in this case this needs to be changed in some other places in BB. Will committing